tasks/main.yml

```diff
@@ -42,3 +42,13 @@
     state: present
     update_cache: yes
     cache_valid_time: 1800
+
+- name: Install docker-compose
+  when: install_docker_compose | bool
```

Comment:
The `|bool` filter should not be necessary, as the variable is already a boolean.